master-pd is not correctly deleted after the test ends

I saw this happen twice: once in "kubemark-100-high-density" suite yesterday, and now in a PR-builder.
The effect is following:
```
Starting master and configuring firewalls
ERROR: (gcloud.compute.disks.create) Some requests did not succeed:
 - The resource 'projects/k8s-jkns-pr-gce/zones/us-central1-f/disks/e2e-gce-agent-pr-50-0-master-pd' already exists

2016/09/09 02:34:43 e2e.go:455: Error running up: exit status 1
```

cc @kubernetes/test-infra-maintainers, assigning @fejta to route to the correct person.

--- a/cluster/gce/util.sh
+++ b/cluster/gce/util.sh
@@ -1183,12 +1183,14 @@ function kube-down() {
   fi
 
   # Delete the master replica pd (possibly leaked by kube-up if master create failed).
-  if gcloud compute disks describe "${REPLICA_NAME}"-pd --zone "${ZONE}" --project "${PROJECT}" &>/dev/null; then
+  # TODO(jszczepkowski): remove also possibly leaked replicas' pds
+  local -r replica-pd="${REPLICA_NAME:-${MASTER_NAME}}-pd"
+  if gcloud compute disks describe "${replica-pd}" --zone "${ZONE}" --project "${PROJECT}" &>/dev/null; then
     gcloud compute disks delete \
       --project "${PROJECT}" \
       --quiet \
       --zone "${ZONE}" \
-      "${REPLICA_NAME}"-pd
+      "${replica-pd}"
   fi
 
   # Delete disk for cluster registry if enabled

